content/docs/user-guide/gems/reference/network/http-requestor.md

```diff
@@ -1,8 +1,165 @@
----
-linkTitle: HTTP Requestor
-title: HTTP Requestor Gem
-description: The HTTP Requestor Gem provides functionality to make asynchronous HTTP/HTTPS requests and return data through a user-provided call back function.
-toc: true
----
-
-The HTTP Requestor Gem provides functionality to make asynchronous HTTP/HTTPS requests and return data through a user-provided call back function.
\ No newline at end of file
+---
+linkTitle: HTTP Requestor
+title: HTTPRequestor Gem
+description: The HTTPRequestor Gem provides functionality to make asynchronous HTTP/HTTPS requests and return data through a user-provided call back function.
+toc: true
+---
+
+The HTTPRequestor Gem provides functionality to make asynchronous HTTP/HTTPS requests and return data through a user-provided call back function.
+
+**Note**  
+This feature is supported only on Windows but could be expanded to other platforms.
+
+## Getting Started
+
+To use the HttpRequestor gem, you must enable it in your project. For more information refer to [Adding and Removing Gems in a Project](/docs/user-guide/project-config/add-remove-gems/).
+
+As HttpRequestor uses pieces of the AWS C++ SDK, you will need to include these pieces in your CMake file:
+
+```
+BUILD_DEPENDENCIES
+        PRIVATE
+            ...
+            Gem::HttpRequestor
+            3rdParty::AWSNativeSDK::Core
+```
+
+## C\+\+ API
```

Comment:
This content will be accepted for now, but should be placed in the header files of the `HttpRequestor` Gem so that we can move from hand-authored API reference on the docs to API reference generated from the headers themselves:

https://o3de.org/docs/api/gems/httprequestor/class_http_requestor_1_1_http_requestor_requests.html